src/main/java/de/qabel/desktop/ui/LayoutController.java

```diff
@@ -228,6 +230,46 @@ private void createButtonGraphics() {
     }
 
 
+    private void setActivityMenu(Label label, ImageView imgView) {
+        label.setVisible(true);
+        imgView.setStyle("-fx-effect: innershadow( gaussian , #222222 , 10 , 10 , 10 , 10 );");
+        if (activeNavItem != null) {
+            activeNavItem.getStyleClass().remove("active");
+        }
+        setBackgroundMenu(label.getId());
+    }
+
+    private void setBackgroundMenu(String menu) {
+        switch (menu) {
+            case "feedbackBackground":
+                inviteBackground.setVisible(false);
+                faqBackground.setVisible(false);
+                inviteButton.setStyle(null);
+                faqButton.setStyle(null);
+                return;
+            case "inviteBackground":
+                feedbackBackground.setVisible(false);
+                faqBackground.setVisible(false);
+                feedbackButton.setStyle(null);
+                faqButton.setStyle(null);
+                return;
+            case "faqBackground":
+                inviteBackground.setVisible(false);
+                feedbackBackground.setVisible(false);
+                inviteButton.setStyle(null);
+                feedbackButton.setStyle(null);
+                return;
```

Comment:
these 3 case blocks are identical, so the switch-case only has 2 different paths. Even those 2 paths are very similar.